File: woodroid/src/main/java/me/gilo/woodroid/models/filters/ProductReviewFilter.java
package me.gilo.woodroid.models.filters;

public class ProductReviewFilter extends ListFilter{

    int[] reviewer;
    int[] reviewer_exclude;
    String[] reviewer_email;

    int[] product;
    String status;

    public int[] getReviewer() {
        return reviewer;
    }

    public void setReviewer(int[] reviewer) {
        this.reviewer = reviewer;

        addFilter("reviewer", reviewer);
    }

    public int[] getReviewer_exclude() {
        return reviewer_exclude;
    }

    public void setReviewer_exclude(int[] reviewer_exclude) {
        this.reviewer_exclude = reviewer_exclude;

        addFilter("reviewer_exclude", reviewer_exclude);
    }

    public String[] getReviewer_email() {
        return reviewer_email;
    }

    public void setReviewer_email(String[] reviewer_email) {
        this.reviewer_email = reviewer_email;

        addFilter("reviewer_email", reviewer_email);
    }

    public int[] getProduct() {
        return product;
    }

    public void setProduct(int[] product) {
        this.product = product;

        addFilter("product", product);
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;

        addFilter("status", status);
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/models/filters/ProductReviewFilter.kt
package me.gilo.woodroid.models.filters

class ProductReviewFilter : ListFilter() {

    internal lateinit var reviewer: IntArray
    internal lateinit var reviewer_exclude: IntArray
    internal lateinit var reviewer_email: Array<String>

    internal lateinit var product: IntArray
    internal lateinit var status: String

    fun getReviewer(): IntArray {
        return reviewer
    }

    fun setReviewer(reviewer: IntArray) {
        this.reviewer = reviewer

        addFilter("reviewer", reviewer)
    }

    fun getReviewer_exclude(): IntArray {
        return reviewer_exclude
    }

    fun setReviewer_exclude(reviewer_exclude: IntArray) {
        this.reviewer_exclude = reviewer_exclude

        addFilter("reviewer_exclude", reviewer_exclude)
    }

    fun getReviewer_email(): Array<String> {
        return reviewer_email
    }

    fun setReviewer_email(reviewer_email: Array<String>) {
        this.reviewer_email = reviewer_email

        addFilter("reviewer_email", reviewer_email)
    }

    fun getProduct(): IntArray {
        return product
    }

    fun setProduct(product: IntArray) {
        this.product = product

        addFilter("product", product)
    }

    fun getStatus(): String {
        return status
    }

    fun setStatus(status: String) {
        this.status = status

        addFilter("status", status)
    }
}
